README.md

```diff
@@ -30,6 +30,24 @@ Interfaz de usuario para la aplicación que permita a los usuarios introducir su
 
 [Historias de usuario](docs/Documentación-1/HUs.md)
 
+## Automatización
+
+### Gestor de versiones
+
+Se han elegido **RVM** como gestor de versiones tras considerar Rbenv como alternativa. El criterio que he seguido ha sido que RVM tiene un mecanismo de instalación de Ruby incorporado, con lo cual lo encuentro más cómodo de usar, a pesar de que Rbenv es más ligero.
+
+Para instalar RVM he seguido las instrucciones de su [página oficial](https://rvm.io/rvm/install).
+
+### Gestor de tareas y dependencias
+
+He elegido **Rake** como gestor de tareas y **Bundler** como gestor de dependencias, ya que no he encontrado ninguna alternativa con la documentación suficientemente clara y estas herramientas están bastante extendidas en la comunidad. 
```

Comment:
Ni siquiera has puesto tres o cuatro herramientas rándom, como dice en [errores frecuentes](https://github.com/JJ/IV-22-23/blob/master/sesiones/semana-08.md#objetivo-3). Si la "documentación suficientemente clara" es un requisito, sigue lo dicho en el enlace anterior sobre los criterios objetivos para elegirlos (por supuesto, tendrás que calificar objetivametne la claridad de la documentación), ese es el punto 3.
Tampoco pones en ningún lado que hayas seguido el punto 2 que se ha comentado.

Reply:
Precisamente porque lo pone en errores frecuentes no puse tres o cuatro herramientas random. Estuve buscando varias alternativas pero no encontré mucho. Voy a intentar expresarlo de otra forma.

Reply:
Pero es que debajo pone cómo se hace correctamente. De verdad que tenéis que entender lo que se busca en el objetivo. ¿De veras creéis que poniendo lo primero que salga en Google como task manager + lenguaje o dependency manager + lenguage con "he elegido" delante y "los demás son *mu malos*" se supera el objetivo?

Reply:
Pero si no he encontrado ninguna alternativa, por ejemplo para el gestor de dependencias, cómo hago lo que viene ahí. No puedo exponer diferentes alternativas y ver cuáles cumplen mis requisitos. 

Para el gestor de tareas sí que vi algunas alternativas (que recuerde una), pero la documentación me parecía liosa (esto sí que puedo clarificarlo en mis criterios).

Reply:
Sigues sin entenderme, y sigues sin entender el proceso. Te remito otra vez a los errores frecuentes.
No puedes buscar "dependency management ruby" y poner lo primero que se encuentra, o lo único que se encuentra, y decir "no hay otra cosa", porque lo que yo quiero no es que busquéis gestores de dependencias de de Ruby. Quiero que aprendáis a tomar decisiones sobre herramientas a incorporar al proyecto.
Así que primero tienes que establecer qué es lo que necesitas de esa herramienta, si es posible con algo específico del proyecto. También buscar cuales son las mejores prácticas en el tema: usar no sé qué fichero de dependencias, o no sé qué otro. *Entonces* es cuando te tienes que poner a buscar y a decidir qué es lo que eliges.
Insisto, *me da igual que en Ruby no haya otro gestor de dependencias*. Lo que quiero es que plantees el problema, establezcas criterios, busques, compares, y si no hay nada, lo digas y lo documentes siguiendo mejores prácticas. Y así con el resto de las herramientas que se van a elegir hasta ahora.
En el caso concreto de Ruby ¿por qué no hay otra alternativa? ¿Es un estándar? ¿Gemfile es un estándar? ¿Dónde dice que sea un estándar? Si Gemfile es un estándar, ¿por qué no hay otras herramientas que lo usen?
De lo que se trata es de que *entendáis vuestras herramientas de trabajo*, no de "poner ahí un gestor de dependencias". No solo de que entendáis las herramientas, sino que comprendáis qué es lo que hay que hacer para conocer las herramientas y que a partir de ahora lo hagáis de forma rutinaria (sí, en en el TFG también, aunque ya sé que no me vais a hacer ni caso)

Reply:
Y sí hay alternativas para el gestor de dependencias. Y para Rake, por supuesto.